demo_usage/convert_tracking_dataset_to_forecasting.py

```diff
@@ -0,0 +1,196 @@
+
+import copy
+import csv
+import pdb
+from pathlib import Path
+from typing import Any, Dict, List, Optional, Tuple
+
+
+import numpy as np
+from argoverse.data_loading.simple_track_dataloader import SimpleArgoverseTrackingDataLoader
+from argoverse.data_loading.object_label_record import json_label_dict_to_obj_record
+from argoverse.utils.se3 import SE3
+from argoverse.utils.transform import quat2rotmat, quat_argo2scipy
+from recordclass import RecordClass
+from scipy.spatial.distance import cdist
+from scipy.spatial.transform import Rotation
+
+
+class Box3d(RecordClass):
+	center: np.ndarray # x, y, z
+	width: float
+	length: float
+	height: float
+	orientation_q: Tuple[float,float,float,float] # qw, qx, qy, qz
+	name: str
+	track_id: str
+	timestamp: int
+	orientation_yaw: Optional[float] = None
+	v0: Optional[Tuple[float,float,float]] = None # vertex 0
+	v1: Optional[Tuple[float,float,float]] = None # vertex 1
+	v2: Optional[Tuple[float,float,float]] = None # vertex 2
+	v3: Optional[Tuple[float,float,float]] = None # vertex 3
+
+
+def rotmat2quat(R: np.ndarray) -> np.ndarray:
+	"""  """
+	q_scipy =  Rotation.from_matrix(R).as_quat()
+	return quat_scipy2argo(q_scipy)
+
+
+def quat_scipy2argo(q_scipy: np.ndarray) -> np.ndarray:
+	"""Re-order Argoverse's scalar-first [w,x,y,z] quaternion order to Scipy's scalar-last [x,y,z,w]"""
+	x, y, z, w = q_scipy
+	q_argo = np.array([w, x, y, z])
+	return q_argo
+
+
+def quaternion_yaw_scipy(q: Tuple[float,float,float,float]) -> float:
+    """ """
+    q_argo = q
+    q_scipy = quat_argo2scipy(q_argo)
+    yaw, _, _ = Rotation.from_quat(q_scipy).as_euler('zyx')
+    return yaw
+
+
+
+
+def remove_duplicate_instances(instances: List[Box3d]) -> List[Box3d]:
```

Comment:
[Nit] Could use a more generic input type for the args if there's no specific need for a `List`. A `Sequence` should be ok here.